coralproject/talk
# Create both a PreMod and a New Queue when stream is switched from one status to another

## What does this PR do?
- Creates both a PreMod and a New Queue when stream is switched from one status to another.

## Edge Case
News Room enables premod every night, and they switch to POST every day. If there are comments with `PREMOD` status we should provide a queue for them. 

## How do I test this PR?
- Enable `PREMOD`
- Post a comment
- Disable `PREMOD`
- Go to Moderation (Both queues should appear and as soon as you reject or accept the PREMOD comment the `PREMOD` queue should disapear)
